-I added Modals to the News section and block level buttons to trigger them, and hyperlinks

Ben8842 opened the pull request:
-The section is now formatted with block level buttons that trigger
modals to show the body of the news article.  hyperlink is also below
the image

Ben8842 committed:
-I added Modal to the News section.
-The section is now formatted with block level buttons that trigger
modals to show the body of the news article.  hyperlink is also below
the image

paragsm1 commented on html/News.html:
```diff
@@ -16,32 +16,130 @@
+            .img-container {
+                text-align: center;
+            }
+        </style>
```
lets move all the styles to .css file

paragsm1 commented on html/News.html:
```diff
@@ -16,32 +16,130 @@
-            <h4 class="newsURL1">Here is the URL: </h4>
-            <h6 class="body1"></h6>
+
+    <h3 style="text-align:center">Latest News</h3>
```
also no in-line styles, move them to .css file and use class or id  to get required styling

Ben8842 committed:
I created the Hot Coin JS function
I created the Hot Coin HTML page
I created the Cold Coin JS function
I created the Cold Coin HTML page
I added a Hot/Cold coin Card for the
main page with graphic (png)

Ben8842 committed:
-Added Sliders to build tool
-Added function to display slider values
-Added dropdown to choose chart type
-modified chart function to grab the drop down values
-fixed the news button titles do display proper news title

paragsm1 commented on html/ColdCoin.html:
```diff
@@ -0,0 +1,134 @@
+</table>
+
+
+
```
Remove extra whitespace
